retriever/bin/retriever.js

```diff
@@ -67,11 +67,8 @@ export default {
     // Timestamp to measure file retrieval performance (from cache and from SP)
     const fetchStartedAt = performance.now()
 
-    const { ownerAddress, pieceRetrievalUrl } = await getOwnerAndValidateClient(
-      env,
-      clientWalletAddress,
-      rootCid,
-    )
+    const { ownerAddress, pieceRetrievalUrl, setId } =
```

Comment:
```suggestion
    const { ownerAddress, pieceRetrievalUrl, proofSetId } =
```

Reply:
Considering the migration from "ProofSetId" to "DatasetId", I find the name "setId" better, because it works for both "ProofSetId" and "DatasetId".

Reply:
it works ok for both but great of neither. `setId` can also refer to other things

Reply:
Fair enough, I'll rename to `proofSetId`.

Reply:
Resolved in [d229586](https://github.com/filcdn/worker/pull/163/commits/d229586d5b8776352863905503b2230c2e00a4a4)